packages/client/src/tests/languages.json

```diff
@@ -1020,8 +1020,6 @@
         "review.rejection.form.commentLabel": "Comments or instructions for health worker to rectify declaration",
```

Comment:
What is this file even :D

Reply:
Seems to be used in `mockOfflineData`.. likely to satisfy Redux tests in client